2019Labs/CustomSecurityContent/documentation/lab2_openscap.adoc

```diff
@@ -177,9 +185,10 @@ image:lab1.2-scapworkbenchscan.png[500,500]
 image:lab1.2-savecustomization.png[300,300]
 
 == Lab 1.3 Security Remediations with OpenSCAP and Ansible
+
 Putting the machine into compliance (for example by changing its configuration) is called *remediation* in the SCAP terminology.
-Remediation changes the configuration of the machine and it is possible that you will lock yourself out or disable important workloads!
-As a result, it is best practice to test the remediation changes before deploying.
+As remediation changes configuration of the machine to restrict it's capabilities, it is possible that you will lock yourself out or disable workloads important to you.
+As a result, it is best practice to test the remediation and it's effects before deploying.
```

Comment:
should be its not it's

Reply:
Removed all changes to this file as per agreement.